packages/server/src/integrations/base/sql.ts

```diff
@@ -646,6 +646,30 @@ class SqlQueryBuilder extends SqlTableQueryBuilder {
     }
     return results.length ? results : [{ [operation.toLowerCase()]: true }]
   }
+
+  async getBulkValues(rowsToInsert: Array<any>, source?: SqlClient) {
+    let finalValues = ""
+    const escapeCharacter = source === SqlClient.POSTGRES ? "E" : ""
```

Comment:
I know there is some common logic here, but it might be easier to implement this per datasource as it is a query type, rather than here, the `base/sql.ts` is more reserved for datasource plus functionality.